cmd/cluster.go

```diff
@@ -143,15 +152,27 @@ func ClusterCommand() cli.Command {
 			{
 				Name:      "export",
 				Usage:     "Export a cluster",
-				ArgsUsage: "[CLUSTERID/CLUSTERNAME...]",
+				ArgsUsage: "[CLUSTERID]",
 				Action:    clusterExport,
+				Flags: []cli.Flag{
+					cli.StringFlag{
+						Name:  "cluster-name, name, n",
+						Usage: "Export by `CLUSTERNAME` instead of CLUSTERID",
+					},
+				},
 			},
 			{
 				Name:      "kubeconfig",
 				Aliases:   []string{"kf"},
 				Usage:     "Return the kube config used to access the cluster",
-				ArgsUsage: "[CLUSTERID CLUSTERNAME]",
+				ArgsUsage: "[CLUSTERID]",
 				Action:    clusterKubeConfig,
+				Flags: []cli.Flag{
+					cli.StringFlag{
+						Name:  "cluster-name, name, n",
+						Usage: "Get kube config for `CLUSTERNAME`",
```

Comment:
Nit: `kubeconfig`. Could be changed in a couple of other places in usages.

Reply:
changed